pkg/pgmodel/metrics/metrics.go

```diff
@@ -58,13 +58,79 @@ var (
 			Help:      "Total number of times leader changed per cluster.",
 		},
 		[]string{"cluster"})
-	MaxSentTimestamp = prometheus.NewGauge(
+	StaleMaxSentTimestamp = prometheus.NewGauge( // This will be deprecated in the renaming PR.
```

Comment:
```suggestion
	 // TODO(Harkishen): This will be deprecated in the renaming PR.
	StaleMaxSentTimestamp = prometheus.NewGauge(
```
nits:
- add `TODO` so the comment is easier to find in IDE (most of them highlight TODO and FIXME statements).
- note who wrote the todo item so it is easier to find someone in the future (without using `git blame`)
- let's keep comments above the code line to adhere to golang comments convention.